templates/user/dashboard/issues.tmpl

```diff
@@ -80,6 +80,10 @@
 								<span class="comment ui right"><i class="octicon octicon-comment"></i> {{.NumComments}}</span>
 							{{end}}
 
+							{{if and .IsTimetrackerEnabled .TotalTimes}}
```

Comment:
Result of `.TotalTimes` be assigned probably to some variable otherwise currently if there is value this query will be executed twice